projects/1.16/assets/buddycards/patchouli_books/buddycard_guidebook/zh_cn/entries/items/grading_sleeve.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "评级卡套",
+  "icon": "buddycards:grading_sleeve",
+  "category": "buddycards:items",
+  "sortnum": 4,
+  "pages": [
+    {
+      "type": "text",
+      "text": "现在我已经提醒过你了！评级卡套不是为随意的巴迪卡牌收藏家准备的。$(br2)评级卡套允许你主手拿着一张未分级的卡，用副手的评级卡套给卡打分。你将获得 5 个等级中的 1 个。"

```

Grading Sleeves are not for the casual Buddycard collector. You have been warned. $(br2)Grading Sleeves allow you to grade cards by holding it in your offhand with an ungraded card in your main hand. It will get 1 of 5 grades.
```suggestion
      "text": "现在我已经提醒过你了！评级卡套不适合为休闲巴迪卡牌收藏家。$(br2)评级卡套允许你主手拿着一张未分级的卡，用副手的评级卡套给卡打分。未分级卡获得5个等级中的1个。"

```